gtsfm/view_graph_estimator/view_graph_estimator_base.py

```diff
@@ -163,6 +163,7 @@ def compute_metrics(
         calibrations: List[Cal3Bundler],
         two_view_reports: Dict[Tuple[int, int], TwoViewEstimationReport],
         view_graph_edges: List[Tuple[int, int]],
+        plots_output_dir: Optional[Path] = None,
```

Comment:
why note just do `plots_output_dir: Path = PLOT_BASE_PATH`, since line 191 effectively does this?